test/recipes/30-test_evp_data/fips_mac.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# Tests start with one of these keywords
+#       Cipher Decrypt Derive Digest Encoding KDF MAC PBE
+#       PrivPubKeyPair Sign Verify VerifyRecover
+# and continue until a blank line. Lines starting with a pound sign are ignored.
+# The keyword Availablein must appear before the test name if needed.
+
+Title = KMAC length failure tests
+
+# Verify that a too short key fails
+FIPSversion = >3.1.3
+MAC = KMAC128
+Key = 01
```

Comment:
Wouldn't this key length fail even for non-fips? `KMAC_MIN_KEY` is set to 4. Shouldn't we use a value that is longer than this but less than `KMAC_MIN_KEY_FIPS`?